### PR TITLE
Minor modifications to interpreted notebook text

### PR DESCRIPTION
**Pull request by georgematheos**
Hi @littleredcomputer ,

I just went through this tutorial.  It's great -- I really enjoyed it and learned a lot about GenJAX from it!

As I went through, I made a few proposals of minor modifications to the text in the notebook, which I thought might be helpful to readers to explain a few concepts in a little bit ore detail.  I think you have a better sense than me what types of audiences will be using this notebook, so feel free to keep or drop any of these proposed edits!

### CODE REVIEW

File: notebooks/concepts/introduction/intro_interpreted.ipynb [hunk 1]
```diff
@@ -62,17 +66,85 @@
...
+    "The `@interpreted` decoration is a bridge between an ordinary Python function and the [Generative Function](https://www.gen.dev/docs/stable/ref/gfi/#Generative-Functions) interface, which is at the heart of the Gen model of probabilistic programming. You can regard the `@` sign as something like the $\\sim$ operator in statistics. (That is, `distribution @ \"b\"` roughly means $b \\sim \\text{distribution}$).  In the above code, `genjax.flip(x) @ b` describes $b$ as a random variable with a $\\text{Bernoulli}(x)$ distribution (like tossing a coin with $x$ as the probability of heads). The GenJAX system uses the string name `\"b\"` to record the name of the random variable; it cannot see the name of the Python variable to which you assigned the value, but of course it's convenient to use the same name in both cases.\n",
     "\n",
-    "The decoration has equipped our function $g$ with the method `simulate`, which will draw a value from the distribution:"
+    "Note that in addition to `genjax.flip`, there is a `genjax.bernoulli` distribution, but its argument is the log probability, rather than the probability.)\n",
```
Comment (littleredcomputer) [inline notebooks/concepts/introduction/intro_interpreted.ipynb]: Or "logits" according to the documentation...is that the same as log probability? I thought it was $\log {p\over{1-p}}$, which isn't the same thing (?)

File: notebooks/concepts/introduction/intro_interpreted.ipynb [hunk 2]
```diff
@@ -302,32 +458,183 @@
...
+    "console.print(observations)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "825f788c",
    "metadata": {},
    "source": [
-    "GenJAX provides numerous sophisticated inference algorithms designed to operate seamlessly with generative functions. For this introduction we will content ourselves with a simple one: Sequential Importance resampling. We won't spend too much time understanding this technique--it is described greater mathematical detail in other notebooks here--but will simply say that we generate a batch of random numbers, score them against the ground truth, and then use the scores as the weight of a random categorical choice (a kind of survival of the fittest), and repeat until, hopefully, the model explains our data."
+    "Currently, the way GenJAX prints out the choicemap does not show the values in it, just the datatypes; but rest assured that each `f32[]` in the above printout corresponds to a value in the observed data:"
```
Comment (littleredcomputer) [inline notebooks/concepts/introduction/intro_interpreted.ipynb:600]: note for the future: someday we should fix it so that if the array is short enough the value appears within the brackets

File: notebooks/concepts/introduction/intro_interpreted.ipynb [hunk 3]
```diff
@@ -342,7 +649,7 @@
     "        tr_lw_pairs = [\n",
     "            model.importance(sub_key, observations, model_args) for sub_key in sub_keys\n",
     "        ]\n",
-    "        lws = [tr_lw[1] for tr_lw in tr_lw_pairs]\n",
+    "        lws = [log_weight for (trace, log_weight) in tr_lw_pairs]\n",
```
Comment (littleredcomputer) [inline notebooks/concepts/introduction/intro_interpreted.ipynb:652]: Thank you, this is much nicer

**Review by littleredcomputer — COMMENTED** [review]
TY George. Just the question about log odds vs. log probability

File: notebooks/concepts/introduction/intro_interpreted.ipynb (hunk 1 above)
Comment (georgematheos) [reply notebooks/concepts/introduction/intro_interpreted.ipynb]: Oh, interesting!  Thanks for reading through in enough detail to catch this!  A quick experiment sampling from `genjax.bernoulli` confirms that it is as you said, log(p/(1-p)).  (At least, genjax.bernoulli(0) seems to be a fair coin flip, and genjax.bernoulli(inf) always returns heads.). I fixed the text in this cell in a new commit.

**Review by littleredcomputer — APPROVED** [review]
LGTM
